app/__init__.py

```diff
@@ -14,30 +18,50 @@
 search_client = SearchClient.create(Config.ALGOLIA_APP_ID, Config.ALGOLIA_API_KEY)
 index = search_client.init_index(Config.INDEX_NAME)
 
+app = Flask(__name__, static_folder='app/static')
+app.config.from_object(Config)
+app.url_map.strict_slashes = False
 
-def create_app(config_class=Config):
-    app = Flask(__name__, static_folder=None)
-    app.config.from_object(config_class)
-    app.url_map.strict_slashes = False
+limiter = Limiter(
+    app,
+    key_func=get_remote_address,
+    default_limits=["200 per day", "50 per hour"]
+)
 
-    Limiter(
-        app,
-        key_func=get_remote_address,
-        default_limits=["200 per day", "50 per hour"]
-    )
+db.init_app(app)
+migrate.init_app(app, db)
 
-    db.init_app(app)
-    migrate.init_app(app, db)
+from app.api import bp as api_bp # noqa
+app.register_blueprint(api_bp, url_prefix='/api/v1')
 
-    from app.api import bp as api_bp
-    app.register_blueprint(api_bp, url_prefix='/api/v1')
+from app.views import bp as view_bp # noqa
+app.register_blueprint(view_bp)
 
-    from app.views import bp as view_bp
-    app.register_blueprint(view_bp)
+from app.errors import bp as error_bp # noqa
+app.register_blueprint(error_bp)
 
-    from app.errors import bp as error_bp
-    app.register_blueprint(error_bp)
 
-    add_health_check(app)
+@app.route("/healthz")
+@limiter.exempt
+def healthz():
+    health = HealthCheck()
+    health.add_section("application", application_data)
+    return health.run()
 
-    return app
+
+@app.route("/environment")
+@limiter.limit("1 per hour")
+def environment():
```

Comment:
Would be cool to get a unit test for this too, but I'm not sure why this was necessary to be honest. I think it might have something to do with that `/metrics` endpoint I mentioned but like I said, I don't recall how all this stuff interacts with each other. Or maybe it was added as just additional information that we could possibly use in the cluster for metrics and such... Anyway, yeah if we're going to have it, might as well get a test around it.

Reply:
Another thing about this.. This is out of scope for this PR, but this endpoint gives out a lot of sensitive information (luckily it masks the credentials, but it gives out all the environment variables and such). We should consider making sure this is only accessible to the cluster and not publicly.

Reply:
then we'll need some kind of access-control plugin for Flask and the implementation of that.  How about we just comment out the `/environment` route for now since it's of no apparent value and a security risk?

Reply:
Yeah we can comment it out for now

Reply:
commented out